tests/integration_test/service_control_failed_request_report_test.go

```diff
@@ -85,6 +85,37 @@ func TestServiceControlFailedRequestReport(t *testing.T) {
 				},
 			},
 		},
+		{
```

Comment:
Can you also add an integration test for a regex-based route with variable binding? The code is split into 2 cases, want to cover both.

Reply:
Done.